icpc-wf/ansible/roles/base_packages/tasks/main.yml

```diff
@@ -93,6 +93,17 @@
     - tig
```

Comment:
Also need to fix the composer as we need the "hip" one which I now locally installed.